multiqc/multiqc.py

```diff
@@ -586,7 +586,7 @@ def run(
         run_modules = [m for m in run_modules if list(m.keys())[0] not in config.exclude_modules]
     if len(run_modules) == 0:
         logger.critical("No analysis modules specified!")
-        sys.exit(1)
+        return {"report": report, "config": config, "sys_exit_code": 1}
```

Comment:
Is there a scenario when it's triggered?

Reply:
```console
❯ multiqc -m fastqc -e fastqc .

  /// MultiQC 🔍 | v1.16.dev0 (f7ad10c)

|           multiqc | Only using modules: fastqc
|           multiqc | Excluding modules 'fastqc'
|           multiqc | No analysis modules specified!
```